star-wars-app/Pages/People/PeopleViewController.swift

```diff
@@ -12,74 +12,33 @@ import RxSwift
 import RxCocoa
 
 class PeopleViewController: UIViewController {
-    let vm = PeopleViewModel()
-    
-    private var people: [Person] = [Person]()
+    private let vm = PeopleViewModel()
+    private let peopleView = PeopleView(frame: UIScreen.main.bounds)
```

Comment:
I suppose it's not a big deal... I haven't considered it before - initilizing view alongside the controller. If you've however initialized it in loadView, then you'd be sure that it's not created until it's needed.